package.json

```diff
@@ -32,22 +32,25 @@
     ]
   },
   "devDependencies": {
+    "@types/escape-html": "^0.0.20",
+    "@types/fs-extra": "^5.0.4",
+    "@types/globby": "^8.0.0",
+    "@types/hash-sum": "^1.0.0",
+    "@types/lru-cache": "^4.1.1",
+    "@types/node": "^10.12.12",
+    "@types/semver": "^5.5.0",
+    "@vue/test-utils": "^1.0.0-beta.29",
     "conventional-changelog-cli": "^1.3.22",
     "eslint": "^4.19.1",
     "eslint-plugin-jest": "^21.15.1",
     "eslint-plugin-vue-libs": "^3.0.0",
+    "inquirer": "^6.2.0",
     "lerna": "^3.6.0",
     "lint-staged": "^7.0.4",
     "minimist": "^1.2.0",
-    "yorkie": "^1.0.3",
-    "inquirer": "^6.2.0",
-    "typescript": "^3.2.2",
-    "@types/node": "^10.12.12",
-    "@types/lru-cache": "^4.1.1",
-    "@types/fs-extra": "^5.0.4",
-    "@types/semver": "^5.5.0",
-    "@types/hash-sum": "^1.0.0",
-    "@types/globby": "^8.0.0",
-    "@types/escape-html": "^0.0.20"
+    "ts-jest": "^24.0.0",
+    "ts-lint": "^4.5.1",
+    "typescript": "^3.3.3",
+    "yorkie": "^1.0.3"
   }
```

Comment:
Don't format package.json, thanks.

Reply:
Didn’t intend to. Had to add some packages to run the tests. Assumed perhaps others may have this dependency issue, hence the commit.